src/Sweden/Mews.Fiscalizations.Sweden/DTOs/CreateActivationRequest.cs

```diff
@@ -0,0 +1,104 @@
+using System.ComponentModel.DataAnnotations;
+using System.Text.Json.Serialization;
+
+namespace Mews.Fiscalizations.Sweden.DTOs;
+
+internal sealed class CreateActivationRequest
+{
+    [Required]
+    [JsonPropertyName("country")]
+    public required string CountryCode { get; init; }
+
+    [Required]
+    [JsonPropertyName("corporateId")]
+    public required string CorporateId { get; init; }
+
+    [Required]
+    [JsonPropertyName("cashRegisterName")]
+    public required string CashRegisterName { get; init; }
+
+    [JsonPropertyName("features")]
+    public string[] Features { get; init; }
+
+    [Required]
+    [JsonPropertyName("controlUnitSerial")]
+    public required string ControlUnitSerial { get; init; }
+
+    [Required]
+    [JsonPropertyName("controlUnitLocation")]
+    public required string ControlUnitLocation { get; init; }
+
+    [JsonPropertyName("validFrom")]
+    public long? ValidFromUnix { get; init; }
+
+    [JsonPropertyName("validTo")]
+    public long? ValidToUnix { get; init; }
+
+    [JsonPropertyName("registrationGeolocation")]
+    public Address Address { get; init; }
+
+    [Required]
+    [JsonPropertyName("applicationNameAndVersion")]
+    public required string ApplicationNameAndVersion { get; init; }
+
+    [Required]
+    [JsonPropertyName("applicationPackage")]
+    public required string ApplicationPackage { get; init; }
+
+    [JsonPropertyName("productionNumber")]
+    public string ProductionNumber { get; init; }
+
+    [JsonPropertyName("installationCreationInfo")]
+    public InstallationCreationInfo InstallationCreationInfo { get; init; }
+
+    [JsonPropertyName("contactInfo")]
+    public ContactInfo ContactInfo { get; init; }
+}
+
+internal sealed class Address
+{
```

Comment:
Not a fan of having this class named Address since companyName is not typically part of an Address but not a blocker. We can maybe rename it later.

Reply:
Yea good point, we can change it later 👍